pkg/doc/verifiable/credential_test.go

```diff
@@ -693,9 +702,10 @@ func TestDownloadCustomSchema(t *testing.T) {
 		}))
 		defer func() { testServer.Close() }()
 
-		_, err := loadCredentialSchema(testServer.URL, &http.Client{})
+		sBytes, err := loadCredentialSchema(testServer.URL, &http.Client{})
```

Comment:
I think "s" as a short form for "schema" here is a bit unintuitive. I would just write out the whole word "schemaBytes" instead.

Reply:
OK